$NetBSD$

Please do NOT touch this patch. It allows gimp-current to be built
on gcc earlier than 3.x  . If you don't know, what it does, why is it
really needed, go and read a bit about differences between standards
supported by gcc 3.x and gcc 2.x, but first - do NOT touch it until it's 
not needed (this has been fixed in gimp-cvs).

	-- dotz@irc.pl

--- app/core/gimpchannel.c.orig	Mon Mar  1 17:34:27 2004
+++ app/core/gimpchannel.c
@@ -640,7 +640,7 @@ gimp_channel_stroke (GimpItem     *item,
   else if (GIMP_IS_PAINT_INFO (stroke_desc))
     {
       GimpImage        *gimage     = gimp_item_get_image (item);
-      GimpPaintInfo    *paint_info = GIMP_PAINT_INFO (stroke_desc);;
+      GimpPaintInfo    *paint_info = GIMP_PAINT_INFO (stroke_desc);
       GimpPaintOptions *paint_options;
       GimpPaintCore    *core;
 
